docs/gl_objects/wikis.rst

```diff
@@ -11,6 +11,9 @@ References
   + :class:`gitlab.v4.objects.ProjectWiki`
   + :class:`gitlab.v4.objects.ProjectWikiManager`
   + :attr:`gitlab.v4.objects.Project.wikis`
+  + :class:`gitlab.v4.objects.GroupWiki`
+  + :class:`gitlab.v4.objects.GroupWikiManager`
+  + :attr:`gitlab.v4.objects.Group.wikis`
 
 * GitLab API: https://docs.gitlab.com/ce/api/wikis.html
 
```

Comment:
Please add link to the API page:

`https://docs.gitlab.com/ee/api/group_wikis.html`